torch/onnx/_internal/exporter.py

```diff
@@ -878,7 +878,14 @@ def __init__(
         self.model_args = model_args
         self.model_kwargs = model_kwargs
 
-        self._assert_fake_tensor_mode()
+        # TODO:Retire FXSymbolicTracer
+        # NOTE: FXSymbolicTracer would fail in this assert, as it does not use `enable_fake_mode`
```

Comment:
what happens if `enable_fake_mode` and `fx_symbolic_graph_extractor` is used at the same time?

Reply:
Unlikely, fx_symbolic_graph_extractor  is not even exposed. I guess it's kept for now only because verification on `enable_fake_mode`.

Reply:
https://github.com/pytorch/pytorch/issues/107714

Reply:
Right, I'm thinking if raising is the right thing to do, if they are not expected to be used together.

Reply:
I think these are all making exceptions for symbolic_tracer? It will make more sense when we retire it.

Reply:
LMK if you would like a follow up on this.